projects/extension/src/background/index.ts

```diff
@@ -24,9 +24,46 @@ export interface Background extends Window {
       listener: (state: ExposedChainConnection[]) => void,
     ) => () => void
     disconnectTab: (tabId: number) => void
+    getLogger: () => LogKeeper
   }
 }
 
+interface LogKeeper {
+  debug: string[]
```

Comment:
```suggestion
  all: string[]
```

Since this also contains non-debug